### ISSUE TITLE
Cloned slide should be tabindex="-1" and aria-hidden="true" when loop="true"

### ISSUE THREAD

**Issue by mrleblanc101** [nolimits4web/swiper#2929]
This is a (multiple allowed):
* [x] bug
* [x] enhancement
* [x] feature-discussion (RFC)

* Swiper Version: 4.0.7
* Platform/Target and Browser Versions: macOS, iOS, Windows, Chrome, Firefox, Safari, IE11, Edge

### What you did
For a11y reason, i added tabindex="0" so screenreader would read my slide content.
Sadly, when swiper 4 clone the slide it also clone those attr. 
Cloned slide should always be hidden to screen reader.
Also it would be nice to disabled autoplay on slide focus natively, I was able to fix this but it really should be build-it.

### Expected Behavior
Cloned slide should always have tabindex="-1" and aria-hidden="true"

### Actual Behavior
Cloned slide have tabindex="0" and no aria-hidden value
